### PR TITLE
INTMDB-490 | Online archive

### PR DESCRIPTION
**Pull request by SuperMohit**
## Description
[(https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Online-Archive)](https://www.mongodb.com/docs/atlas/reference/api-resources-spec/#tag/Online-Archive)

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

File: cfn-resources/online-archive/Makefile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+.PHONY: build test clean
+tags=logging callback metrics scheduler
+cgo=0
+goos=linux
+goarch=amd64
+
+ldXflags=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=info
+ldXflagsD=github.com/mongodb/mongodbatlas-cloudformation-resources/util.defaultLogLevel=debug
```
Comment (govardhanpagidi) [inline cfn-resources/online-archive/Makefile:8]: change info level to warning, we changed info level to warning

File: examples/online-archive/online-archive.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        "ClusterName": "CFNTest-Cluster-001",
+        "CollName": "rombo-bongo",
+        "Criteria": {
+          "Type": "DATE",
+          "DateField": "created",
+          "DateFormat": "ISODATE",
+          "ExpireAfterDays": "5"
+        },
+        "DbName": "testdb",
+        "ProjectId": "625454459c4e6108393d650d",
```
Comment (govardhanpagidi) [inline examples/online-archive/online-archive.json:22]: Remove ProjectID

**Review by govardhanpagidi — APPROVED** [review]
LGTM!

File: cfn-resources/online-archive/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+new MongoDB::Atlas::SearchIndex Resource Type into your AWS Region. Note, this command uses the
+default AWS region.
+
+```bash
+TAGS=logging make
+cfn submit --verbose --set-default
+```
+## CloudFormation Examples
+
+Please see the [test/inputs_1_create.template.json](test/inputs_1_create.template.json) for example resource for example resource.
```
Comment (Zuhairahmed) [inline cfn-resources/online-archive/README.md:44]: Broken link, please fix

File: cfn-resources/online-archive/README.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+Installation currently requires the follow 2 steps to build and then submit/register the
+new MongoDB::Atlas::SearchIndex Resource Type into your AWS Region. Note, this command uses the
+default AWS region.
+
+```bash
+TAGS=logging make
+cfn submit --verbose --set-default
+```
+## CloudFormation Examples
```
Comment (Zuhairahmed) [inline cfn-resources/online-archive/README.md:42]:
```suggestion
## CloudFormation Example
```

**Review by Zuhairahmed — APPROVED** [review]
Few minor changes, not blocking. Thanks @SuperMohit!

**Review by swattyT — APPROVED** [review]

**Pull request merged**